StructureDefinition/NHSDigital-Immunization.StructureDefinition.xml

```diff
@@ -227,95 +225,76 @@
         <expression value="identifier.exists().not() or identifier.where(system='https://fhir.hl7.org.uk/Id/gmc-number').exists().not() or (identifier.where(system='https://fhir.hl7.org.uk/Id/gmc-number').exists()  and identifier.where(system='https://fhir.hl7.org.uk/Id/gmc-number').value.matches('^[C]{1}[0-9]{7}$'))" />
-        <source value="https://fhir.nhs.uk/StructureDefinition/NHSDigital-Immunization" />
       </constraint>
       <constraint>
         <key value="usercode-gphc" />
         <severity value="error" />
         <human value="GPHC must be of the format NNNNNNN" />
         <expression value="identifier.exists().not() or identifier.where(system='https://fhir.hl7.org.uk/Id/gphc-number').exists().not() or (identifier.where(system='https://fhir.hl7.org.uk/Id/gphc-number').exists()  and identifier.where(system='https://fhir.hl7.org.uk/Id/gphc-number').value.matches('^[0-9]{7}$'))" />
-        <source value="https://fhir.nhs.uk/StructureDefinition/NHSDigital-Immunization" />
       </constraint>
       <constraint>
         <key value="usercode-hcpc" />
         <severity value="error" />
         <human value="HCPC must be of the format AANNNNNN" />
         <expression value="identifier.exists().not() or identifier.where(system='https://fhir.hl7.org.uk/Id/hcpc-number').exists().not() or (identifier.where(system='https://fhir.hl7.org.uk/Id/hcpc-number').exists()  and identifier.where(system='https://fhir.hl7.org.uk/Id/hcpc-number').value.matches('^[A-Z]{2}[0-9]{6}$'))" />
-        <source value="https://fhir.nhs.uk/StructureDefinition/NHSDigital-Immunization" />
       </constraint>
       <constraint>
         <key value="usercode-din" />
         <severity value="error" />
         <human value="DIN format must be NNNNNN" />
         <expression value="identifier.exists().not() or identifier.where(system='https://fhir.hl7.org.uk/Id/din-number').exists().not() or (identifier.where(system='https://fhir.hl7.org.uk/Id/din-number').exists()  and identifier.where(system='https://fhir.hl7.org.uk/Id/din-number').value.matches('^[0-9]{6}$'))" />
-        <source value="https://fhir.nhs.uk/StructureDefinition/NHSDigital-Immunization" />
       </constraint>
       <constraint>
         <key value="role-sds-role-profile" />
         <severity value="error" />
         <human value="SDS Role Profile Id must be 12 digits" />
         <expression value="identifier.where(system='https://fhir.nhs.uk/Id/sds-role-profile-id').exists().not() or (identifier.where(system='https://fhir.nhs.uk/Id/sds-role-profile-id').exists()  and identifier.where(system='https://fhir.nhs.uk/Id/sds-role-profile-id').value.matches('^[0-9]{12}$'))" />
-        <source value="https://fhir.nhs.uk/StructureDefinition/NHSDigital-PractitionerRole" />
+        <source value="https://fhir.nhs.uk/StructureDefinition/England-PractitionerRole" />
       </constraint>
       <mustSupport value="true" />
     </element>
-    <element id="Immunization.performer.actor.type">
```

Comment:
Removal of these constraints seems to go outside the scope of canonical URL renaming, was this removed because it was redundant or was there another reason?